src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/DiagnosticTelemetryEvent.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+
+namespace NuGet.PackageManagement.Telemetry
+{
+    public class DiagnosticTelemetryEvent : TelemetryEvent
+    {
+        public DiagnosticTelemetryEvent(string eventName) :
+            base(eventName)
+        {
+            this[nameof(NuGetVersion)] = VSTelemetryServiceUtility.NuGetVersion.Value;
```

Comment:
Isn't this unnecessary? 
The VS version is contained in every telemetry event.

Reply:
Or do you want it in order to make the analysis easier?

Reply:
To make analysis easier.  Mapping VS version to NuGet version is work.